packages/vaadin-notification/src/vaadin-notification.js

```diff
@@ -475,6 +477,40 @@ class NotificationElement extends ThemePropertyMixin(ElementMixin(PolymerElement
       }
     }
   }
+
+  static show(contents, duration, position) {
```

Comment:
Common JS API would be to have an `options` object parameter instead of separately `this` and `that`. Would that make more sense so we can add more options later?

Reply:
I think that would be a good alternative.

I'll check back if we want to do this, I'm not sure how much planning went into the acceptance criteria, or if it was just adopted from this PR.

Reply:
OK, let's go ahead with the options object parameter.

Reply:
Done